Source/options.cpp

```diff
@@ -973,21 +973,28 @@ GraphicsOptions::GraphicsOptions()
               { ScalingQuality::AnisotropicFiltering, N_("Anisotropic") },
           })
     , integerScaling("Integer Scaling", OptionEntryFlags::CantChangeInGame | OptionEntryFlags::RecreateUI, N_("Integer Scaling"), N_("Scales the image using whole number pixel ratio."), false)
-    , vSync("Vertical Sync",
+#endif
+    , frameRateControl("Vertical Sync",

```

Comment:
The ini setting name should probably also change to "Frame Rate Control"?

```suggestion
    , frameRateControl("Frame Rate Control",
```

Reply:
cant belive i missed that again :D